### DISCUSSION TITLE
analog devices linux kernel (2019_r2) is not accessible.

### DISCUSSION THREAD

**Discussion by murinrtc** [community/community#112314] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i have been facing the following issue since 3 months, on cloning the analog devices linux kernel (2019_r2). please provide solution for this. 

**git clone -b 2019_R2 https://github.com/analogdevicesinc/linux.git** 
**Cloning into 'linux'...
remote: Enumerating objects: 10911136, done.
remote: Counting objects: 100% (6320/6320), done.
remote: Compressing objects: 100% (2131/2131), done.
error: RPC failed; curl 56 GnuTLS recv error (-9): A TLS packet with unexpected length was received.
fatal: The remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed**

Also, my major problem is in petalinux build. 
error message is as follows. 

**linux-xlnx-4.19-adi_2019_R2+gitAUTOINC+89a181de04-r0 do_fetch: Fetcher failure for URL: 'git://github.com/analogdevicesinc/linux.git;protocol=https;branch=2019_R2'. Unable to fetch URL from any source.**

please provide the solution as soon as possible. i am struck in my project application and in urgent need of help

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @murinrtc! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by murinrtc** [reply]
Would anyone please guide on this? I need to resolve it as earlier as possible. 

Thanks.

**Comment by murinrtc** [comment]
why no one gives reply here? I am struck in this and need to resolve the problem. @ghostinhershell
